.github/workflows/actions/compile-gh-pages/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Compile GitHub Pages
+description: Compiles content for GitHub Pages and uploads it to the artifact cache
+
+inputs:
+  pr-number:
+    type: number
+  repository:
+    default: 'solana-web3.js'
+    type: string
```

Comment:
I think it'd be useful to add a comment here that the inputs are only used for the `REACT_EXAMPLE_APP_BASE_PATH` and therefore for new web3js

It took me a while to figure out what's going on when the legacy job calls this without any inputs